packages/atlas-service/src/main.ts

```diff
@@ -46,7 +47,9 @@ const redirectRequestHandler = oidcServerRequestHandler.bind(null, {
 /**
  * https://www.mongodb.com/docs/atlas/api/atlas-admin-api-ref/#errors
  */
-function isServerError(err: any): err is { errorCode: string; detail: string } {
+function isServerError(
+  err: any
+): err is { error: number; errorCode: string; detail: string } {
```

Comment:
If we are changing the assertion, the check should also be updated, otherwise the assertion don't match what is being returned